src/parser/jobs/brd/modules/AdditionalStats.js

```diff
@@ -100,25 +102,55 @@ export default class AdditionalStats extends Module {
 			if (event.type.match(/^(apply|remove|refresh)(de)?buff(stack)?$/)) {
 				// In case it's a status on the selected player
 				if (event.targetID === this.combatants.selected.id && event.ability) {
-					this._player.statuses[event.ability.guid] = event.type.startsWith('apply') || event.type.startsWith('refresh')
+					const player = this._player
+					player.statuses[event.ability.guid] = {
+						isActive: event.type.startsWith('apply') || event.type.startsWith('refresh'),
+					}
+
+					// If the current status is a crit modifier, we add the strength onto it to use later
+					const critModifier = CRIT_MODIFIERS.find(cm => cm.id === event.ability.guid)
+					if (critModifier) {
+
+						// If it's a spear card, we get the modifier from the event, thank's to arcanum
+						if (critModifier.id === STATUSES.THE_SPEAR.id) {
```

Comment:
nit; you're using `critModifier.strength` in both cases, can move the base modifier out of the conditional and just have the conditional case for spear funtimes.